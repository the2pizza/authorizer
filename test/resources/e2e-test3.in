{"account": {"active-card": false, "available-limit": 750}}
{"transaction": {"merchant": "Burger King", "amount": 20, "time": "2019-02-13T10:00:00.000Z"}}
{"transaction": {"merchant": "Habbib's", "amount": 90, "time": "2019-02-13T11:00:00.000Z"}}