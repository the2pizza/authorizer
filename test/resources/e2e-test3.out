{"account":{"active-card":false,"available-limit":750},"violations":[]}
{"account":{"active-card":false,"available-limit":750},"violations":[]}
{"account":{"active-card":false,"available-limit":750},"violations":[]}